packages/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -139,28 +138,19 @@ public function get_at_rule() {
 		 * @return string
 		 */
 		public function get_css( $should_prettify = false, $indent_count = 0 ) {
-			$rule_indent                = $should_prettify ? str_repeat( "\t", $indent_count ) : '';
-			$nested_rule_indent         = $should_prettify ? str_repeat( "\t", $indent_count + 1 ) : '';
-			$declarations_indent        = $should_prettify ? $indent_count + 1 : 0;
-			$nested_declarations_indent = $should_prettify ? $indent_count + 2 : 0;
-			$suffix                     = $should_prettify ? "\n" : '';
-			$spacer                     = $should_prettify ? ' ' : '';
+			$rule_indent         = $should_prettify ? str_repeat( "\t", $indent_count ) : '';
```

Comment:
Just reverting to what is was before https://github.com/WordPress/gutenberg/pull/58867